src/java/io/bazel/rulesscala/specs2/FilteredSpecs2ClassRunner.scala

```diff
@@ -28,15 +30,21 @@ class FilteredSpecs2ClassRunner(testClass: Class[_], testFilter: Pattern)
 
   private def createFilteredDescription(specStructure: SpecStructure, ee: ExecutionEnv): Description = {
     val descTree = createDescriptionTree(ee).map(_._2)
-//    descTree.toTree.bottomUp {
-//      (description: Description, children: Stream[Description]) =>
-//        children.filter(matchingFilter).foreach {
-//          child => description.addChild(child)
-//        }
-//        description
-//    }.rootLabel
-    descTree.getLabel
 
+    def bottomUp[A, B](t: Tree[A], f: ((A, Iterable[B]) => B)): Tree[B] = {
```

Comment:
had to copy code and modify slightly to use generic Iterable to ensure it builds on both S2 and S3